hadoop-ozone/dist/src/main/compose/common/hadoop-test.sh

```diff
@@ -16,12 +16,9 @@
 # limitations under the License.
 
 export COMPOSE_FILE=docker-compose.yaml:../common/hadoop.yaml
-export HADOOP_MAJOR_VERSION=${HADOOP_VERSION%%.*}
-
-export SECURITY_ENABLED=false
+export HADOOP_MAJOR_VERSION=3
```

Comment:
This can also be `unused`? Since it is set in the for-loop. And both Hadoop 2 and 3 are tested.

Reply:
`unused` doesn't work here, we get the following error:

```
$ ./test-hadoop.sh
Couldn't find env file: .../hadoop-ozone/dist/target/ozone-1.4.0-SNAPSHOT/compose/common/hadoopunused.conf
```